speechbrain/lobes/models/huggingface_transformers/speecht5.py

```diff
@@ -0,0 +1,483 @@
+    *args : tuple
+        Variable-length positional arguments. Each argument can be of any type.
+    **kwargs : dict
+        Variable-length keyword arguments. Each argument name is a string, and
+        each value can be of any type.
+
+    Example
+    -------
+    >>> model_hub = "microsoft/speecht5_asr"
+    >>> save_path = "tmp"
+    >>> model = SpeechT5ForASR(model_hub, save_path=save_path) # doctest: +SKIP
+    """
+
+    def __init__(
+        self,
+        source: str,
+        save_path: str = "",
+        cache_dir: str = "",
+        freeze: bool = False,
+        freeze_encoder: bool = False,
+        freeze_feature_extractor: bool = False,
+        sampling_rate: int = 16_000,
+        encoder_only: bool = False,
+        output_attentions: bool = False,
+        output_all_hiddens: bool = False,
+        device=None,
+        *args,
+        **kwargs,
+    ) -> None:
+
+        if encoder_only and freeze:
+            freeze_encoder = True
+
+        if encoder_only and freeze_encoder:
+            freeze = True
+
+        super().__init__(
+            source=source,
+            save_path=save_path,
+            freeze=freeze,
+            cache_dir=cache_dir,
+            device=device,
+            *args,
+            **kwargs,
+        )
+
+        self.sampling_rate = sampling_rate
+        self.encoder_only = encoder_only
+        self.freeze_encoder = freeze_encoder
+        self.freeze_feature_extractor = freeze_feature_extractor
+        self.output_attentions = output_attentions
+        self.output_all_hiddens = output_all_hiddens
+        self.generation_config = None
+
+        self.load_tokenizer(source=source, **kwargs)
+
+        config = SpeechT5Config.from_pretrained(
+            source, cache_dir=cache_dir, **kwargs
+        )
+        self.config = self.override_config(config)
+
+        if self.encoder_only:
+            logger.warning(
+                f"{self.__class__.__name__}: Using an encoder-only SpeechT5 for speech to text."
+            )
+            del self.model.speecht5.decoder
+            del self.model.text_decoder_postnet
+
+        if self.freeze_feature_extractor:
+            self.model.freeze_feature_encoder()
+
+    def forward(self, wav, decoder_input_ids=None):
+        """
+        One forward step of the SpeechT5 model ** using the manually implemented methods ** by calling
+        the `forward_encoder()` and `forward_decoder()` methods.
+
+        Arguments
+        ----------
+            wav (raw waveform to be fed to the model.): torch.Tensor or numpy.ndarray
+            decoder_input_ids (If using a decoder, those would be the tokenized text targets.): list
+
+        Returns
+        -------
+            Encoder-only: Either the encoder hidden layers (last or all internal stacked if `self.output_all_hiddens` is `True`)
+            Encoder-Decoder: The encoder hidden states (see encoder-only), the logits, and the attentions
+        """
```

Comment:
there's an extra indent in the docstring args :p please remove it

Reply:
I had to add it to fix the Flake8 error for this particular docstrings xD 
Remember  ?